x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts

```diff
@@ -494,7 +494,7 @@ export const threshold = t.intersection([
   thresholdField,
   t.exact(
     t.partial({
-      cardinality: t.union([t.array(thresholdCardinalityField), t.null]),
+      cardinality: t.array(thresholdCardinalityField),
```

Comment:
@madirey were we using `null` here to represent some meaning different from `undefined`?

Reply:
I think that was just left over from a development fixup... should be fine to get rid of it now that we've released.